docs/07_Installing_Essential_Cluster_Services.md

```diff
@@ -214,7 +217,7 @@ pvc-9de54e35-b9c2-48fa-b92a-c365ef3dff5e   8Gi        RWO            Delete
 Finally, let's see them in the NFS exported directory:
 
 ```
-$ ls -l nfs-pvs
+$ sudo ls -l nfs-pvs
```

Comment:
I don't like the fact that `nfs-pvs` is in current user's home directory, and they don't have access to it.